react/features/welcome/components/SideBarItem.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Linking, Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+
+import { Icon } from '../../base/font-icons';
+import { translate } from '../../base/i18n';
+
+type Props = {
+
+    /**
+     * The i18n label of the item.
+     */
+    i18Label: string,
+
+    /**
+     * The icon of the item.
+     */
+    icon: string,
+
+    /**
+     * The function to be invoked when the item is pressed
+     * if the item is a button.
+     */
```

Comment:
This says "if the item is a button." Do instances with only url and not onPress display like links?

Reply:
They look the same, but yes, they act as a link. In that case, the _onOpenURL function is invoked